apps/style/code-studio/petition.scss

```diff
@@ -0,0 +1,116 @@
+.petition_block {
```

Comment:
@tim-dot-org would love your style eyes on this ... I tried to get away with using Flex box as much as possible, but still ended up using a lot of other styling. Couldn't find similar styles from our common style sheet

Reply:
+1! @tim-dot-org I'd also love a vote of confidence in the idea of using an scss file here -- I forget what the latest best practices are, and I seem to be writing all my CSS as inline styles on react components still.

Reply:
Yes, agreed. I started with inline styles, that got wild fast, and then switch to Tim's approach for reference guide styling. But not sure what our best practices are and would like to veer towards that if possible

Reply:
I don't know that we have any "official" best practices or processes to get us there, but this was the most recent proposed doc on the topic: https://docs.google.com/document/d/1Y3uK_iYMhTUaCI6yIDwOAMprIJCuzXSs2fECQggwp60/edit#heading=h.htf3pg55q2kt
(here, radium is the inline styles, and using an scss file is most of the way (but not entirely) towards css modules)

I prefer a separate file since its easier to do things that css natively supports, like cascading and pseudo selectors.

Reply:
Got it, thanks. I remember reading that now 😅 . Pseudo selectors on hovering over the link is what finally did it for me to switch